examples/ckks/bootstrapping/main.go

```diff
@@ -24,20 +24,23 @@ func main() {
 	var plaintext *ckks.Plaintext
 
 	// Bootstrapping parameters
-	// Four sets of parameters (index 0 to 3) ensuring 128 bit of security
-	// are available in github.com/tuneinsight/lattigo/v3/ckks/bootstrap_params
+	// 2xFour sets of parameters (index 0 to 3) ensuring 128 bit of security
```

Comment:
Eight, ... index 0 to 7.

Reply:
The two set of four parameters are in different structs.

Reply:
I see. Then it should be better explained that four sparse-key sets and four dense-key sets are available at...